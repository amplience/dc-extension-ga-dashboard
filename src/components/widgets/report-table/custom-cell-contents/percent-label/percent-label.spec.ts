import { render } from '@testing-library/svelte';
import PercentLabel from './percent-label.svelte';

describe('PercentLabel', () => {
  it('should render a PercentLabel component', () => {
    const { container } = render(PercentLabel, { value: 25 });

    expect(container.firstChild).toMatchSnapshot();
  });
});
