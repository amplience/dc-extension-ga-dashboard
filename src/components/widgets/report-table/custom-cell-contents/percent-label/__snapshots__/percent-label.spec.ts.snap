// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PercentLabel should render a PercentLabel component 1`] = `
<div>
  <div
    class="svelte-1av7bio"
  >
    <div
      class="percent-bar svelte-1av7bio"
    >
      <div
        class="percent-bar-marker svelte-1av7bio"
        style="width: 25%;"
      />
       
      <div
        class="percent-value svelte-1av7bio"
      >
        25
        %
      </div>
    </div>
  </div>
</div>
`;
